helm/memcached-operator/helm-charts/memcached/README.md

```diff
@@ -2,7 +2,7 @@
 
 > [Memcached](https://memcached.org/) is an in-memory key-value store for small chunks of arbitrary data (strings, objects) from results of database calls, API calls, or page rendering.
 
-Based on the [memcached](https://github.com/helm/charts/tree/master/stable/memcached) chart from the [Helm Charts](https://github.com/helm/charts) repository.
+Based on the [memcached](https://github.com/bitnami/charts/tree/master/incubator/memcached) chart from the [Bitnami Charts](https://github.com/bitnami/charts) repository.
```

Comment:
```suggestion
Based on the Bitnami [memcached chart](https://github.com/bitnami/charts/tree/master/bitnami/memcached).
```

Reply:
i is not added by us. it is from the Memcached chart. So we should not change.

Reply:
The current link doesn’t exist. This link points to the bitnami memcached chart’s new location.